src/main/java/org/logstash/plugins/inputs/http/HttpServerHandler.java

```diff
@@ -0,0 +1,53 @@
+package org.logstash.plugins.inputs.http;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.*;
+import io.netty.handler.codec.compression.DecompressionException;
+import io.netty.handler.codec.http.*;
```

Comment:
nitpick: most style guides i have seen recommend against wildcard imports. in IntelliJ just set the min number to 99 in the Java formatter settings.